jest.server.json

```diff
@@ -2,13 +2,18 @@
   "coverageDirectory": "./build2/reports/coverage",
   "restoreMocks": true,
   "moduleNameMapper": {
-    "^OktaAuth(.*)$": "<rootDir>/jquery/$1"
+    "^OktaAuth$": "<rootDir>/lib/server/serverIndex"
   },
   "testMatch": [
     "**/test/spec/*.js"
   ],
   "testPathIgnorePatterns": [
-    "./test/spec/oauthUtil.js"
+    "./test/spec/fingerprint.js",
```

Comment:
why ignore those files?

Reply:
The server version (Node) only support AuthN APIs (similar to the AuthN java sdk), but no fingerprint, webfinger, TokenMAnager, OIDC flows, etc..

Reply:
ah, make sense. I conflicts with the other one `jest.browser.json`